portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/displaypagetemplate/DisplayPageTemplatesWithItemPreview.testcase

```diff
@@ -480,4 +480,116 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-131040. Only the public vocabularies are available in preview item selector."
+    @priority = "4"
+    test OnlyPublicVocabulariesPreviewSelector {
```

Comment:
Should start with a verb. Like ViewOnlyPublicVocabulariesAvailableToItemSelector